framework/math/quadratures/angular/lebedev_orders/lebedev_order003.csv

```diff
@@ -0,0 +1,6 @@
+1.0,0.0,6.123233995736766e-17,0.166666666666667

```

Comment:
I assume you computed these somehow and you got these machine zero numbers. You probably want to filter those and have exact `0.` instead.